ui-admin/src/study/participants/survey/SurveyEditView.tsx

```diff
@@ -1,10 +1,61 @@
 import React from 'react'
 import { Survey, SurveyResponse } from 'api/api'
+import DocumentTitle from 'util/DocumentTitle'
+
+import _cloneDeep from 'lodash/cloneDeep'
+import { Enrollee, PagedSurveyView, useTaskIdParam } from '@juniper/ui-core'
+import { StudyEnvContextT } from 'study/StudyEnvironmentRouter'
+import { Store } from 'react-notifications-component'
+import { failureNotification, successNotification } from 'util/notifications'
+import { usePortalLanguage } from 'portal/usePortalLanguage'
 
 /** allows editing of a survey response */
-export default function SurveyEditView({ response, survey }: {response: SurveyResponse, survey: Survey}) {
-  console.log(`not implemented ${response.surveyId} ${survey.stableId}`)
+export default function SurveyEditView({ studyEnvContext, response, survey, enrollee, adminUserId, onUpdate }: {
+  studyEnvContext: StudyEnvContextT, response?: SurveyResponse,
+  survey: Survey, enrollee: Enrollee, adminUserId: string, onUpdate: () => void
+}) {
+  const { defaultLanguage } = usePortalLanguage()
+  const taskId = useTaskIdParam()
+  if (!taskId) {
+    return <span>Task Id must be specified</span>
+  }
+  const workingResponse = response ? _cloneDeep(response) : makeEmptyResponse(enrollee, survey, adminUserId)
+  const studyEnvParams = {
+    studyShortcode: studyEnvContext.study.shortcode,
+    envName: studyEnvContext.currentEnv.environmentName,
+    portalShortcode: studyEnvContext.portal.shortcode
+  }
   return <div>
-    Not yet implemented
+    <DocumentTitle title={`${enrollee.shortcode} - ${survey.name}`}/>
+    <div>
+      <PagedSurveyView
+        studyEnvParams={studyEnvParams}
+        form={survey}
+        enrollee={enrollee}
+        response={workingResponse}
+        selectedLanguage={defaultLanguage.languageCode}
+        adminUserId={adminUserId}
+        onSuccess={() => {
+          onUpdate()
+          Store.addNotification(successNotification('Response saved'))
+        }}
+        onFailure={() => Store.addNotification(failureNotification('Response could not be saved'))}
+        updateProfile={() => {}} //eslint-disable-line @typescript-eslint/no-empty-function
```

Comment:
I think you can avoid the typescript warning by having it return null or true, or some other simple value
```suggestion
        updateProfile={() => null} 
```

Reply:
you an also have a commment on the inside 
```
() => {
// todo
}
```